xml/System/Type.xml

```diff
@@ -2100,16 +2103,7 @@ The `filter` argument can be a custom delegate of type <xref:System.Reflection.M
 >  You cannot omit parameters when looking up constructors and methods. You can only omit parameters when invoking.  
   
  If the current <xref:System.Type> represents a constructed generic type, this method returns the <xref:System.Reflection.ConstructorInfo> with the type parameters replaced by the appropriate type arguments. If the current <xref:System.Type> represents a type parameter in the definition of a generic type or generic method, this method always returns `null`.  
-  
-   
-  
-## Examples  
- The following program obtains the type of `MyClass1` class, gets the <xref:System.Reflection.ConstructorInfo> object matching the specified binding flags, and displays the signature of the constructor.  
-  
- [!code-cpp[Type_GetConstructor2#1](~/samples/snippets/cpp/VS_Snippets_CLR/Type_GetConstructor2/CPP/type_getconstructor2.cpp#1)]
```

Comment:
I noticed that for this page the samples were not the same, so we should keep it in the member itself and simply convert the original format to the new syntax instead.

Reply:
Does that mean removing the new syntax from the member group and adding them back in member with new syntax.

Reply:
Correct.